ddtrace/profiling/exporter/pprof.pyx

```diff
@@ -88,7 +87,7 @@ class _PprofConverter(object):
             return self._locations[(filename, lineno, funcname)]
         except KeyError:
             if funcname is None:
-                real_funcname = _line2def.filename_and_lineno_to_def(filename, lineno)
+                real_funcname = "<unknown function>"
```

Comment:
So,  theoretically, this branch should never be hit and is more like a safety net?

Reply:
Yes, with the current collector set this is never hit.